presto-product-tests/bin/run_on_docker.sh

```diff
@@ -121,8 +121,8 @@ trap terminate INT TERM EXIT
 
 # display how test environment is configured
 environment_compose config
-
-environment_compose up --no-color &
+SERVICES=$(environment_compose config --services | grep -vx 'application-runner\|.*-base')
```

Comment:
What's special about `application-runner` and `.*-base`. Doesn't this make the behavior brittle (e.g., if we some of the services)?

Reply:
`application-runner` fails unless command is vhttps://github.com/prestosql/presto/blob/6f92ef2dde02264eca03bda9eddcf2c936fa2986/presto-product-tests/conf/docker/common/standard.yml#L48
`java-8-base` exits cleanly, immediately (whatever the command actually is)

To include them (aka be able to not exclude them)  I'd need to define sth like `sleep 999d` for each. Then they would not exit, but they will be running unnecessarily.